dspace-api/src/test/java/org/dspace/app/csv/CSVMetadataImportReferenceIT.java

```diff
@@ -175,12 +253,13 @@ public void testMultiRowNameRef() throws Exception {
     @Test
     public void testSingleUUIDReference() throws Exception {
         context.turnOffAuthorisationSystem();
-        Item person = ItemBuilder.createItem(context, col1)
-                                 .withRelationshipType("Person")
-                                 .build();
+        WorkspaceItem workspaceItem = workspaceItemService.create(context, col1, false);
+        Item person = installItemService.installItem(context, workspaceItem);
+        itemService.addMetadata(context, person, "relationship", "type", null, null, "Person");
+        itemService.update(context, person);
```

Comment:
Again, remove this new code and revert to using the `ItemBuilder` class.  We require Builders.